lib/transactional.pm

```diff
@@ -176,3 +177,28 @@ sub ensure_rollback_service_not_running {
         $output =~ '^(start|running)$' ? sleep 10 : last;
     }
 }
+
+
+# Verify that essential snapshots cannot be deleted, see https://jira.suse.com/browse/SLE-3804
+sub check_undel_snaps {
+    # Create some snapshots with a transactional update snapshot in the middle.
+    record_info "Check undel", "Check that essential snapshots cannot be deleted, creates snapshots and installs a package";
+    assert_script_run("snapper create");
+    trup_call "ptf install" . rpmver('security');
+    assert_script_run("snapper create");
+
+    my $last_snapshot        = +(split /\n/, script_output('snapper list --disable-used-space'))[-1];
```

Comment:
According to docs, "Unary "+" has no effect whatsoever, even on strings. It is useful syntactically for separating a function name from a parenthesized expression that would otherwise be interpreted as the complete list of function arguments'

Therefore, as you don't have any function on the left, let's remove it.

Reply:
btw, you can `pop` instead of using index.

Reply:
pop says "Experimental pop on scalar is now forbidden" in this case, it would require one extra line. I removed those +.

Reply:
I was referring to do the pop over the list returned by split, it should work in theory.

Reply:
yes, that's what I tried, it returned that message in openqa. Not enough "strict" maybe.